modules/ROOT/pages/6.1-release-notes.adoc

```diff
@@ -224,11 +224,11 @@ This caused {productname} to present as if an undo option was available when the
 This update fixes this bug. When the Comments plugin loads, the comment model data is no longer placed into an undo level and no Undo actions will present as available until there are actual changes available to undo.
 
 
-==== Opening the comments sidebar via an editor command did not respect the `skip_focus` argument
+==== Opening the comments sidebar via an editor command did not respect the `grabFocus` argument
 
-If the Comments sidebar was opened with an editor command and `skip_focus` was set to true — `editor.execCommand("ToggleSidebar", false, "showcomments", { skip_focus: true });` — the sidebar still stole focus.
+If the Comments sidebar was opened with an editor command and `grabFocus` was set to true — `editor.execCommand("'tc-open-comment'", false, "showcomments", { grabFocus: true });` — the sidebar still stole focus.
```

Comment:
```suggestion
If the Comments sidebar was opened with an editor command and `grabFocus` was set to true — `editor.execCommand("tc-open-comment", false, "showcomments", { grabFocus: true });` — the sidebar still stole focus.
```

How about `the sidebar still had the focus` instead of `the sidebar still stole focus`? As my understanding is the editor should have the focus instead of the comments sidebar when `grabFocus` is true